filters-share/specific_ELEMHIDE.txt

```diff
@@ -22,7 +22,9 @@ dcinside.com##div[class$="banner"]
 toomics.com,nongmin.com,ilbe.com##div[class$="-banner"]
 ! ##div[class^="banner"]
 m.edaily.co.kr,ichannela.com,m.news1.kr##div[class^="banner"]
-! 
+!
+m.10000recipe.com##.view_sp_banner
```

Comment:
Can you provide the URL?
If it is private, please email me.

Reply:
https://m.10000recipe.com/recipe/step.html?seq=6937381&step=0